lib/suspenders/generators/runner_generator.rb

```diff
@@ -0,0 +1,75 @@
+require_relative "base"
+
+module Suspenders
+  class RunnerGenerator < Generators::Base
+    def procfile
+      copy_file "Procfile", "Procfile"
+    end
+
+    def sample_env
+      copy_file "sample_env", ".sample.env"
+    end
+
+    def copy_sample_env
+      if bin_setup_is_ruby?
+        insert_into_file "bin/setup",
+          %{  puts "\\n== Copying sample env =="\n  system! 'cp -i .sample.env .env'\n\n},
+          before: %{  puts "\\n== Preparing database =="}
+      elsif bin_setup_mentions_ci?
+        insert_into_file "bin/setup", %{  cp -i .sample.env .env\n},
+          after: %{if [ -z "$CI" ]; then\n}
+      else
+        append_to_file "bin/setup",
+          %{\nif [ -z "$CI" ]; then\n  cp -i .sample.env .env\nfi}
+      end
+    end
+
+    def update_readme
+      configure = <<~README
```

Comment:
I _think_ GitHub will syntax highlight Ruby heredocs delimited by language names.

Are you willing to try that out? 

I think that would involve using `MD` or `MARKDOWN` instead of `README`

Reply:
Tried `MARKDOWN`, but no dice. That said, it's a better name.